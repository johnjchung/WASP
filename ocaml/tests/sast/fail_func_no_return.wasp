Func int squareFloat (int x, int y) {
	return 345;
	int i = 11;
}