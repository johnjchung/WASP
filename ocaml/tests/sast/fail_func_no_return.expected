Fatal error: exception Type_checking.NoReturnErr("No return statement found for function squareFloat()")
